### PR TITLE
GITHUB TEMPLATE - incentive to try nightly and search for opened issue

### PR DESCRIPTION
**Pull request by MikeZeDev**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-connector-website-not-working.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ assignees: ''
...
 Yes/No
 
-**Did you [try Nightly Build](https://github.com/manga-download/hakuneko/releases) ?**
-Yes/No
+**Make sure to [try Nightly Build](https://github.com/manga-download/hakuneko/releases) .**
+Nightly build contains latest fixes, maybe your problem is already solved.
 
-**Is the website of the connector working properly / are you able to see the manga within your browser**
-Yes/No
+**Make sure to [try Nightly Build](https://github.com/manga-download/hakuneko/releases) .**
```
Comment (Sheepux) [inline .github/ISSUE_TEMPLATE/1-connector-website-not-working.md]: do not remove the question about "is the website working", you're creating duplicates
Also i was planning on moving the issues to forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by Sheepux** [comment]
I've pushed the github form version in your branch, can you check it ?

File: .github/ISSUE_TEMPLATE/1-connector-website-not-working.md (hunk 1 above)
Comment (MikeZeDev) [reply .github/ISSUE_TEMPLATE/1-connector-website-not-working.md]: Removal was a mistake indeed

**Comment by MikeZeDev** [comment]
> I've pushed the github form version in your branch, can you check it ?

Seems good to me, nice work

**Review by Sheepux — APPROVED** [review]

**Pull request merged**